interface/VideoDecoderDefs.h

```diff
@@ -83,6 +84,7 @@ typedef struct {
     int32_t bitrate;
     int32_t framerateNom;
     int32_t framerateDenom;
+    uint32_t fourcc;
```

Comment:
Bump version is boring and maybe conflict with other commits. Since we do not know which commit will merge first.
Let us focus on code first. I will bump up the version when this merged.